src/rust/engine/process_execution/src/lib.rs

```diff
@@ -234,6 +249,16 @@ pub struct MultiPlatformExecuteProcessRequest(
   pub BTreeMap<(Platform, Platform), ExecuteProcessRequest>,
 );
 
+impl MultiPlatformExecuteProcessRequest {
+  pub fn user_facing_name(&self) -> Option<String> {
+    let mut buf = String::new();
+    for (_, epr) in self.0.iter() {
```

Comment:
Rather than ignoring the platform, how about making this look like:
`Linux: Exec[foo bar], Darwin: Exec[foo, baz]`?

Reply:
This could potentially get very long, not allowing you to see both things. Imagine that the console breaks in the "|":
`Linux: Exec[super long command that actually overflows the "|"console by itself...]`

We could add something in the beginning to indicate that it's multiplatform, so that even if it's truncated the users know there's two of them.
Or, we could prioritize which one we show first based on the local platform, maybe.
Or both :)